bibliopixel/animation/sequence.py

```diff
@@ -37,15 +38,6 @@ def addAnim(self, anim, amt=1, fps=None, max_steps=0, until_complete=False,
     def preRun(self, amt=1):
         self.animIndex = -1
 
-    def run(self, amt=1, fps=None, sleep_time=0, max_steps=0, until_complete=False,
```

Comment:
From my email: 
All that does is throw out everything passed in except for fps and until_complete and the run the rest with some hard coded values. This is required for keeping AnimationQueue API compatible with all other animations because users could pass other values in but nothing other than those really make sense....
The reason that fps is even used at all is that if you, for example, add all your animations with no specific FPS you can then specify FPS to the queue and all sub animations should run at that rate.
Note... We MUST keep AnimationQueue/Sequence API compatible with all other animations. It IS an animation like any other. Without that it cannot easily be run through things like PixelWeb.